### ISSUE TITLE
background-image in css

### ISSUE THREAD

**Issue by EffEss** [pinkynrg/convergence2.0#46]
Can you please add an option to change background-image to default or something circuit-ey like:
https://www.dropbox.com/s/vi4cdca8vd2shwg/circuit-background.png?dl=0

**Comment by pinkynrg** [comment]
yes I intended to do that as soon as possible. Here I have a nice one made by raul where there are also tiny LGV and similar. 

http://convergence.provvedo.com/images/style/background.png

anyway I'll make it customisable for sure.
